### PR TITLE
Randomtrackingdict module code quality and unit test

### PR DESCRIPTION
**Pull request by navjotcis**
1) written test case for randomtrackingdict module
2) moved randomtrackingdict.py outside the network directory
3) replaced the print statement of filesystem.py file with logger for fixing some of the issues of #1373

### CODE REVIEW

**Comment by g1itch** [comment]
>     2. moved randomtrackingdict.py outside the network directory

Why? Where it's used outside of the `network` module?

**Comment by PeterSurda** [comment]
@g1itch It isn't used outside, but there is no network-specific code in it. More importantly, it is difficult to have unit tests executed correctly for stuff under `network`. Probably still some leftover circular imports or maintaining compatibility for running it without setuptools. I think once we have a sufficient number of distro-independent packaging procedures available (mainly appimage, perhaps snap), then we can revisit how to fix this properly and perhaps it can be moved to network again.

File: src/storage/filesystem.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from os import listdir, makedirs, path, remove, rmdir
 from threading import RLock
 
+from debug import logger
```
Comment (g1itch) [inline src/storage/filesystem.py:10]: bad practice, please read the `debug` doc

**Comment by g1itch** [comment]
> @g1itch It isn't used outside, but there is no network-specific code in it. More importantly, it is difficult to have unit tests executed correctly for stuff under `network`.

Maybe, but I still don't understand why `from pybitmessage.network import ..` not works, but `from pybitmessage import ..` works.

**Comment by PeterSurda** [comment]
> Maybe, but I still don't understand why ...

I also don't fully understand it, my guess is that the migration to setuptools isn't finished and some leftover code that expects code to be in the current directory rather than as a module. It needs to be fixed eventually but for now we'll have to work around it.

**Comment by g1itch** [comment]
> > Maybe, but I still don't understand why ...
> 
> I also don't fully understand it, my guess is that the migration to setuptools isn't finished and some leftover code that expects code to be in the current directory rather than as a module. It needs to be fixed eventually but for now we'll have to work around it.

Oh, I see another import from outer package not working without pathmagick:
```
>>> from pybitmessage.network import randomtrackingdict
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "pybitmessage/network/__init__.py", line 4, in <module>
    from addrthread import AddrThread
  File "pybitmessage/network/addrthread.py", line 6, in <module>
    import state
ImportError: No module named state
```

File: src/storage/filesystem.py (hunk 1 above)
Comment (PeterSurda) [reply src/storage/filesystem.py:10]: I don't understand, what's wrong here? This is like it's used everywhere else in the project.

**Review by PeterSurda — COMMENTED** [review]
Looks ok.

File: src/storage/filesystem.py (hunk 1 above)
Comment (g1itch) [reply src/storage/filesystem.py:10]: Another import from outer package. When I refactored the logger I left the [note](https://pybitmessage.rtfd.io/en/v0.6/autodoc/pybitmessage.debug.html):
> The old form: from debug import logger is also may be used, but only in the top level modules.

Comment (PeterSurda) [reply src/storage/filesystem.py:10]: I see.

Comment (navjotcis) [reply src/storage/filesystem.py]: I have replaced debug with logging import

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**